.github/ISSUE_TEMPLATE/patch_release.md

```diff
@@ -11,67 +11,71 @@ assignees: ''
+  - **Update `OrchardCore.Commons.props`**: Set `<VersionPrefix></VersionPrefix>` to the new version you're preparing for release.
+  - **Update Module Versions**: Modify `src/OrchardCore/OrchardCore.Abstractions/Modules/Manifest/ManifestConstants.cs` to reflect the new version.
+  - **Release Notes**: Finalize the release notes in the documentation, including:
+      - Highlights and goals of the release.
+      - Prerequisites for running the new version.
+      - Upgrade steps and any breaking changes.
+  - **Update Documentation Navigation**: Add the release notes page to `mkdocs.yml` navigation and remove it from `not_in_nav`.
+  - **Version Mentions**: Update all references to the new version throughout the documentation, including:
+    - [Status in the root README](https://docs.orchardcore.net/en/latest/#status)
+    - CLI templates and commands.
+    - Relevant guides, such as the [Creating a new decoupled CMS Website](https://docs.orchardcore.net/en/latest/guides/decoupled-cms/) guide.
+- [ ] Create a **version PR** titled `Release <version number` (e.g., `Release 2.1.1`) from the version branch (e.g., `release/2.1.1`) into the release branch (e.g., `release/2.1`)
+- [ ] In GitHub, manually run the `Preview - CI` workflow on your branch (NOT `main`). This will release a new preview version on Cloudsmith for testing.
 
 ## Step 3: Validation
 
-1. **Check Functionality**: Update [`OrchardCore.Samples`](https://github.com/OrchardCMS/OrchardCore.Samples) to the latest preview version generated in the previous step. Ensure the samples work as expected.
-2. **Test Guides**: Test the following guides with NuGet packages from the CloudSmith feed:
-   - [Creating a modular ASP.NET Core application](https://docs.orchardcore.net/en/latest/guides/create-modular-application-mvc/)
-   - [Creating an Orchard Core CMS website](https://docs.orchardcore.net/en/latest/guides/create-cms-application/)
-   - [Creating a new decoupled CMS Website](https://docs.orchardcore.net/en/latest/guides/decoupled-cms/)
+- [ ] **Check Functionality**: Update [`OrchardCore.Samples`](https://github.com/OrchardCMS/OrchardCore.Samples) to the latest preview version generated in the previous step (just change the `OrchardCoreVersion` property in the root `Directory.Build.props` file). Ensure the samples work as expected.
+- [ ] **Test Guides**: Test the following guides with NuGet packages from the Cloudsmith feed:
+  - [Creating a modular ASP.NET Core application](https://docs.orchardcore.net/en/latest/guides/create-modular-application-mvc/)
+  - [Creating an Orchard Core CMS website](https://docs.orchardcore.net/en/latest/guides/create-cms-application/)
+  - [Creating a new decoupled CMS Website](https://docs.orchardcore.net/en/latest/guides/decoupled-cms/)
+- [ ] If everything looks good and all checks pass, merge the version PR.
 
-## Step 4: Create New Release
+## Step 4: Creating the New Release
```

Comment:
Create New Release is better for me. But whatever you like

Reply:
The template previously mixed the imperative mood of verb with nouns in titles (_create_ the new release/_align_ branches vs _backporting_ pull requests/code and documentation _updates_). Now it's all nouns. I'm fine with either of these, but I'd stick to one.

Reply:
I am okay with sticking with one.

Reply:
That's the case now.